doc/1.0-roadmap.md

```diff
@@ -0,0 +1,110 @@
+# client-rust 1.0 roadmap
+
+The client-rust project complements the TiKV ecosystem by providing a reliable Rust client. It can be a foundation for implementing clients in other languages.
+
+The document describes our plan to push client-rust towards its general availability (1.0).
+
+## Deliverable
+
+client-rust 1.0 that
+- enables new users to start using it in 10 mins
+- has been heavily tested under simulated and real workload
+- is easy to use, debug and seek help from documentation
+- is performant
+- supports the most-wanted features: GC, async commit, etc.
```

Comment:
"GC": does it mean the API to manually trigger GC?

Reply:
Yes. In the current implementation, we only provide a `gc` function to update GC safepoint. Because client-rust serves as a library, users must manually manage the GC and trigger it appropriately. 

Things might change if the GC mechanism changes (https://github.com/tikv/rfcs/pull/65).